crane-cloud/frontend
# Fix No Resource Messages

# Description

Modifies the no resource messages for the app list, project list and database list making them more relevant with a create button  and also integrates css modules for all changed files.

## Type of change

Please delete options that are not relevant.


## Trello Ticket ID

https://trello.com/c/09Zi0qC9

## How Can This Been Tested?

Run this branch locally and create a new project with no existing apps and be able to view the new no-resource message and do the same for the database lists, whereby if you have no existing databases, the no-resource message will appear.

## Checklist:


# Screenshots

Before
![image](https://user-images.githubusercontent.com/63339234/125929145-5cd530bc-c3a8-4d5e-89d8-d27dce6b690c.png)

After
![image](https://user-images.githubusercontent.com/63339234/125929765-ea1faf11-7cfe-44c8-be60-b8e70340d4fd.png)
